README.md

```diff
@@ -19,5 +23,10 @@ Then run `carthage update`.
 
 ## Usage
 
+To use DesignableKit all you need to do after you have imported the framework is to set the class to `IBButton` for a `UIButton` and `IBView` for a `UIView`. Then you need to set the module to DesignableKit.
+
+![attributes inspector](Images/identity-inspector.png)
+
+Once the above is done you should see a lot more attributtes to under the attributtes inspector that you you can manipulate. The changes should be shown immediately in interface builder if everything was done right. 
```

Comment:
I would prefer "Once the above is done you should see a lot more attributes under the attribute inspector that you you can manipulate. The changes should be shown immediately in the interface builder if everything was done correctly."

Reply:
Thanks for the feedback and it is done :)